tests/test_benchmark.py

```diff
@@ -52,12 +52,15 @@ class Benchmarker(object):
   def __init__(self, name):
     self.name = name
 
-  def bench(self, args, output_parser=None, reps=TEST_REPS):
+  def bench(self, args, output_parser=None, reps=TEST_REPS, expected_output=None):
     self.times = []
     self.reps = reps
     for i in range(reps):
       start = time.time()
       output = self.run(args)
+      if expected_output is not None and expected_output not in output:
+        raise ValueError('Incorrect benchmark output:\n' + output)
```

Comment:
can use `self.assertContained` like the main test suite, it will show a nice diff on errors.

Reply:
I can't use `assertContained` because this is not in a test. This is in the benchmarker class.

Reply:
Oh, ok.